caffe2/operators/layer_norm_op.cc

```diff
@@ -186,16 +186,21 @@ to the end.)
 
 // Register layer norm with c10
 namespace {
+struct State final : public c10::KernelState {
+    at::optional<at::Tensor> scale = at::nullopt;
```

Comment:
use .defined instead of optional?

Reply:
In this diff, this is just copying the state from another place to here, not changing what the state is. A future diff will change the state to store the caffe2 operator instance as written in the dispatcher API design doc. So `scale` here will be gone.